### PR TITLE
creating i3 cheatsheet for ddg

### PR DESCRIPTION
**Pull request by goromlagche**
# Goodie Pull Request Template                                                                                                                                            
                                                                                                                                                                          
Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of
our GitHub pull request and complete as appropriate.                                                                                                                      
                                                                                                                                                                          
**What does your Instant Answer do?**                                                                                                                                     
This is a cheat-sheet for [i3](http://i3wm.org/) window manager.                                                                                                                              
                                                                                                                                                                          
**What problem does your Instant Answer solve (Why is it better than organic links)?**                                                                                    
Its a cheatsheet so it gives a glance of the regularly used key bindings. This cheatsheet is more updated, like the keybinding to open launch menu mentioned on the link  "Mod1+v" does not work anymore, it is updated to "Mod1+d". Most of the i3 documentation is not updated properly.                                                           
                                                                                                                                                                          
**What is the data source for your Instant Answer? (Provide a link if possible)**                                                                                         
It comes from http://i3wm.org/docs/refcard.pdf, this is the official resource.                                                                                            
                                                                                                                                                                          
**Why did you choose this data source?**                                                                                                                                  
The link provided is the official resource.                                                                                                                               
                                                                                                                                                                          
**Are there any other alternative (better) data sources?**                                                                                                                
Not something I can find.                                                                                                                                                 
                                                                                                                                                                          
**What are some example queries that trigger this Instant Answer?**                                                                                                       
i3 cheatsheet.                                                                                                                                                            
                                                                                                                                                                          
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**                                                                       
People who use a window manager on a linux system.                                                                                                                
                                                                                                                                                                          
**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**                                                                      
Not really.                                                                                                                                                               
                                                                                                                                                                          
**Which existing Instant Answers will this one supercede/overlap with?**                                                                                                  
Nothing I can find.                                                                                                                                                       
                                                                                                                                                                          
**Are you having any problems? Do you need our help with anything?**                                                                                                      
Review the code.                                                                                                                                                          
                                                                                                                                                                          
**Where did you hear about DuckDuckHack? (For first time contributors)**                                                                                                  
Bangalore Quack&Hack meetup group (http://www.meetup.com/Quack-Hack-Bangalore/)                                                                                           
                                                                                                                                                                          
**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**                                                                                        

-----

IA Page: https://duck.co/ia/view/i3_cheat_sheet

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
Hi @goromlagche, thanks for this! Would you mind undoing your changes to the [Goodies PR template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_goodie.md) and instead filling it in in your first comment, here?

**Comment by goromlagche** [comment]
@MariagraziaAlastra sorry for the mess. 
p.s. shouldn't this be in the CI check (not changing the pr template file)

**Comment by MariagraziaAlastra** [comment]
@goromlagche No problem! Anyway we still want to be able to modify the template, e.g. we might want to add/remove questions :)

**Comment by goromlagche** [comment]
@MariagraziaAlastra  Though I do not know about your internal setup or travis, but there should be some way, you know like only people with enough permissions can modify that file, something like that. 
Anyways, you guys have a pretty simplified process to write a plugin (Y)

File: share/goodie/cheat_sheets/i3.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
+    "name": "i3 window manager",
+    "id": "i3_cheat_sheet",
+    "description": "Window Manager",
+    "metadata": {
+        "sourceName": "i3CheatSheat",
+        "sourceUrl": "http://i3wm.org/docs/refcard.pdf"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/i3.json]: Looks like they support HTTPS, let's update the link :+1:

File: share/goodie/cheat_sheets/i3.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        "Changing container modes",
+        "Opening other application",
+        "Using workspaces",
+        "Restarting i3 inplace",
+        "Exiting i3"
+    ],
+    "sections": {
+        "Moving Around": [{
+            "val": "Open new Terminal",
+            "key": "[Mod1] + [Enter]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/i3.json]: Sorry about the confusion, but we just revides our syntax docs and have decided to lose the `+` between combined keys.

Could you remove them from here?

```
[Mod1] + [Enter] => [Mod1] [Enter]
```

**Comment by goromlagche** [comment]
@moollaza [fixed](https://github.com/goromlagche/zeroclickinfo-goodies/commit/0095d1f97443085937cdf4b207505e6bd8e6a3f0). I totally forgot to check for "https" link. Also I added ascii chracters for navigation keys [here](https://github.com/duckduckgo/zeroclickinfo-goodies/commit/9ed4c4b2420fc3123ff1803b2da7d394a13da844)

**Comment by moollaza** [comment]
@goromlagche sorry for the delay, this LGTM :+1:

**Comment by moollaza** [comment]
@goromlagche congrats! It's live: https://duckduckgo.com/?q=i3+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**